bpf/bpf_host.c

```diff
@@ -1126,9 +1123,12 @@ int to_host(struct __ctx_buff *ctx)
 #endif /* ENABLE_HOST_FIREWALL */
 
 out:
-	if (IS_ERR(ret))
+	if (IS_ERR(ret)) 
 		return send_drop_notify_error(ctx, srcID, ret, CTX_ACT_DROP,
 					      METRIC_INGRESS);
+	else if (!traced)
```

Comment:
nit: I realize that I suggested this, but it seems to me that `else` is probably not needed here, because the `if` branch returns. Can you please remove the `else` here?

Reply:
> nit: I realize that I suggested this, but it seems to me that `else` is probably not needed here, because the `if` branch returns. Can you please remove the `else` here?

Yeah I realized that too
working on that, thank you for your suggestions!